roles/openshift_health_checker/test/search_journalctl_test.py

```diff
@@ -0,0 +1,199 @@
+import pytest
+import search_journalctl
+import sys
+
+from time import time
+
+
+SAVED_TIMESTAMPS = list()
+
+if sys.version_info > (3, 0):
+    RE_ERROR_UNEXPECTED_END_REGEXP = "unterminated character set"
+else:
+    RE_ERROR_UNEXPECTED_END_REGEXP = "unexpected end of regular expression"
```

Comment:
Hmm, looks catchy -- why would we be checking error message text based depending on Python version?! Hmm, better not.